frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -174,7 +163,7 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
     (snapshotClass: VolumeSnapshotClassKind) => provisioner?.includes(snapshotClass?.driver),
     [provisioner],
   );
-  const vscList = vscLoaded ? vscObj.items : [];
+  const vscList = vscLoaded && vscObj ? vscObj.items : [];
```

Comment:
This probably can be

```suggestion
  const vscList = vscObj?.items || : [];
```

Reply:
done